src/sqlancer/questdb/gen/QuestDBAlterIndexGenerator.java

```diff
@@ -29,7 +30,13 @@ public static SQLQueryAdapter getQuery(QuestDBGlobalState globalState) {
         sb.append(" ");
 
         sb.append("ALTER COLUMN ");
-        String columnName = table.getFreeColumnName();
+
+        // We should always choose column with SYMBOL type
+        QuestDBColumn columnWithSymbolType = table
+                .getRandomColumnOrBailout(c -> c.getType().toString().equals("SYMBOL"));
```

Comment:
Here, rather than a string comparison, could we directly compare the type? This has the advantage that the comparison is type-safe. Besides that, the PR looks good to me.